nodeprobe/nodeprobe.go

```diff
@@ -114,15 +114,12 @@ func (p *Prober) probe(ctx context.Context) {
 	p.ready.Store(allNodesReady.Load())
 
 	if !p.ready.Load() {
-		p.logger.Debug("not all nodes are ready")
+		p.logger.Warn("not all nodes are ready")
 		if h := p.unreadyHandler.Load(); h != nil {
 			(*h)()
 		}
 		return
 	}
-
-	p.logger.Info("all nodes are ready")
```

Comment:
Why don't we need this log?

Reply:
We should refrain from documenting normal operations; instead, focus on logging occurrences of issues or anomalies, just as it will do if they are not ok.

for example, this log is written every minute and clutters the output